doc/conf.py

```diff
@@ -0,0 +1,68 @@
+# -*- coding: utf-8 -*-
+# Project information
+project = u'Mantella'
+copyright = u'2016, Sebastian Niemann, Oskar Pusz, Daniel Kiechle, Wolf Bock von Wülfingen and Philipp Schiffmann'
+author = u'Mantella team and contributors'
+github_url = 'https://github.com/SebastianNiemann/Mantella'
+
+version = u'0.50'
+release = u'0.50.0'
```

Comment:
0.61.0

Can't this be build from `version` with the patch version (`.0`) added to it?